_episodes/02-REPL.md

```diff
@@ -32,7 +33,7 @@ The first thing they try is to perform basic arithmetic operations
 {: .output}
 
 That works as expected.
-It is also possible to bind names to values via the assignment operator `=`, which makes it easier to refer to them later on.
+It is also possible to ~~bind~~ allot names to values via the assignment operator `=`, which makes it easier to refer to them later on.
```

Comment:
Actually creating a binding is the official terminology for this.
Do you think allot is easier to understand for newcomers?

Reply:
You are right. The technical term is variable binding.
My suggestion:

Variable binding are done using assignment operator `=`. Allotting names to variables makes it easier to refer to them later on.

____________
I use the term allot often in my teaching. It has a hidden message, that the name once chosen is reserved.

Reply:
>  that the name once chosen is reserved.

But that isn't true in julia code. You can bind the same name to different values at different points in the code.
Actually I would avoid the term `variable` since it gives the wrong mental model of "the value is bound to the name" which is the other way around.
Which makes it hard to understand something like
```julia
a = [1,2]
b = a
b[2] = 3
print(a) # gives [1, 3]
```

Reply:
I quickly searched for the definition on Julia and quote it:

" A variable, in Julia, is a name associated (or bound) to a value. It's useful when you want to store a value (that you obtained after some math, for example) for later use."
https://docs.julialang.org/en/v1/manual/variables/

Albeit variable bind makes a lot of sense now. We should perhaps use this definition.

Reply:
```suggestion
It is also possible to create a variable by binding a name to a value via the assignment operator `=`, which makes it easier to refer to them later on.
```
Are you okay with that?

Reply:
+1